src/CmsTextBlockBundle/Bridge/Chameleon/Migration/Script/update-1637069390.inc.php

```diff
@@ -0,0 +1,9 @@
+<h1>Build #1637069390</h1>
+<div class="changelog">
+    - ref #736: add info about template change
+</div>
+<?php
+
+TCMSLogChange::addInfoMessage('The text block rendering now passes all callTimeVars to the wysiwyg rendering.
```

Comment:
placeholders?

Reply:
Yes. The variables are passed to the renderer to replace the placeholders with the name of the variables. They are optional placeholder contents.

Reply:
hmmmm yes and no. Because the variable name was not changed and is callTimeVars. I've changed the text to make it more clear.